README.md

```diff
@@ -160,6 +164,111 @@ It can be useful to add additional gems to the bundle. This can be done w/o edit
 
 **NOTE:** Do not add gems to the gemspec nor Gemfile.  When you add to the knapsack Gemfile/gemspec, when you bundle, you'll update the Hyku Gemfile; which will mean you might be updating Hyku prime with knapsack installation specific dependencies.  Instead add gems to `./bundler.d/example.rb`.
 
+## Features
+
+This HykuKnapsack includes several custom features for consortium-based multitenancy:
+
+### Consortium-Based Tenant Configuration
+
+This knapsack overrides Hyku's default profile loading behavior. Instead of all tenants using the same metadata profile, tenants can now have their own consortium-specific profiles based on their `part_of_consortia` setting.
```

Comment:
```suggestion
This knapsack overrides Hyku's default profile loading behavior. Instead of all tenants using the same default metadata profile, tenants can now have their own consortium-specific default profiles based on their `part_of_consortia` setting.
```